docs/ddl-of-carbondata.md

```diff
@@ -495,6 +497,14 @@ CarbonData DDL statements are documented here,which includes:
      TBLPROPERTIES('LOAD_MIN_SIZE_INMB'='256')
      ```
 
+   - ##### Range Column
```

Comment:
Better to use the sentence "Only one column can be configured".
During data loading, you can use "global_sort_partitions" or "scale_factor" to avoid generating small files

Reply:
fixed